util/expensivequery/memory_usage_alarm.go

```diff
@@ -143,7 +146,6 @@ func (record *memoryUsageAlarm) doRecord(memUsage uint64, instanceMemoryUsage ui
 			err := os.Remove((*filename)[0])
 			if err != nil {
 				logutil.BgLogger().Error("remove temp files failed", zap.Error(err))
-				return
```

Comment:
why remove this?

Reply:
If some files are removed manually, it will always log the error and try to remove it again.  I think that it should skip the file if there is some remove error.